streamlit_agent/simple_feedback.py

```diff
@@ -0,0 +1,66 @@
+from langchain.agents import initialize_agent, AgentType
+from langchain.callbacks import StreamlitCallbackHandler, LangChainTracer
+from langchain.chat_models import ChatOpenAI
+from langchain.tools import DuckDuckGoSearchRun
+from langsmith import Client
+import streamlit as st
+
+st.set_page_config(page_title="LangChain: Simple feedback", page_icon="🦜")
+st.title("🦜 LangChain: Simple feedback")
+
+openai_api_key = st.sidebar.text_input("OpenAI API Key", type="password")
+langsmith_api_key = st.sidebar.text_input("LangSmith API Key", type="password")
+ls_tracer = None
+if langsmith_api_key:
+    project = st.sidebar.text_input("LangSmith Project", value="default")
+    ls_client = Client(api_url="https://api.smith.langchain.com", api_key=langsmith_api_key)
+    ls_tracer = LangChainTracer(project_name=project, client=ls_client)
```

Comment:
I was having trouble setting the right environment variables in-line based on dynamic user input, so switched to the explicit tracer setting method to make the deployed app works.

If you set the ENV values in secrets.toml as shown in the README example, it works just fine without any special setup here - tracing happens automatically and you can init the client with no args